### PR TITLE
add package plus, minus, x, search, check icons

### PR DESCRIPTION
**Pull request by Reund0**
adding package group icons  including:

package-plus icon,

package-plus

package-minus icon,

package-minus

package-search icon,

package-search

package-x icon,

package-x

package-check icon.

package-check

### CODE REVIEW

**Review by bennetfabian — APPROVED** [review]

**Review by ericfennis — CHANGES_REQUESTED** [review]
@Reund0 Nice icons! These are really welcome.

Did you read the  [Lucide Icon Design guide](https://github.com/lucide-icons/lucide/blob/master/docs/ICON_DESIGN_GUIDE.md)?
We like to see 1px padding of the 24x24 pixel grid. I see there is some place to move the icon a bit to the right to center it.

image
Also maybe you can try to increase the sub-icons in the corner a bit if it fits. 
If you need any help let me know.

Other things we like to see is that the icon if formatted like all the other icons. See [Global Attributes](https://github.com/lucide-icons/lucide/blob/master/docs/ICON_DESIGN_GUIDE.md#global-attributes) in icon design guide, or look a an other icon in the icons directory.

And at least please add some tags to the `tags.json`  located in the root directory. This improves the search engine on the website. So other devs and designers will find your beautiful icons!

**Comment by mittalyashu** [comment]
The package search icon seems a bit congested.

@Reund0 Can you try increase the size of circle of magnifying glass?

**Comment by Reund0** [comment]
> The package search icon seems a bit congested.
> 
> @Reund0 Can you try increase the size of circle of magnifying glass?

Is it okay to not have 2px spacing?

**Comment by mittalyashu** [comment]
@Reund0 

> Is it okay to not have 2px spacing?

Ah! I see, what you mean. That will be bit tricky 😞 

@ericfennis Do you think it would be good idea to keep the package icon edge to edge that way the children icon (not just search other icons as well) will get more space.

**Comment by ericfennis** [comment]
@mittalyashu I think it's better to leave it centered, that matches better it the current package icon if used together.

**Comment by ericfennis** [comment]
@Reund0 Can you also increase the size of the X a bit in the package-x icon?

**Comment by Reund0** [comment]
> @Reund0 Can you also increase the size of the X a bit in the package-x icon?

I increased its size from 4x4 to 4.5x4.5 but again is it okay to ignore 2px spacing?

File: icons/package-search.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16.5 9.4 7.55 4.24M21 10V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l2-1.14" />
+  <path d="M3.29 7 12 12l8.71-5M12 22V12" />
+  <circle cx="18.58" cy="15.58" r="2.18" />
+  <path d="m22 19-1.87-1.87" />
+</svg>
```
Comment (ericfennis) [inline icons/package-search.svg:16]:
```suggestion
  <path d="M21 10V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l2-1.14" />
  <path d="M16.5 9.4 7.55 4.24" />
  <path d="M12 12v10M3.29 7 12 12 3.29 7ZM12 12l8.71-5L12 12Z" />
  <circle cx="18.5" cy="15.5" r="2.5" />
  <path d="M20.27 17.27 22 19" />
</svg>

```

File: icons/package-x.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16.5 9.4 7.55 4.24M21 10V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l2-1.14" />
+  <path d="M3.29 7 12 12l8.71-5M12 22V12m5.5 1.5L22 18m-4.5 0 4.5-4.5" />
+</svg>
```
Comment (ericfennis) [inline icons/package-x.svg:14]:
```suggestion
  <path d="M21 10V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l2-1.14" />
  <path d="M16.5 9.4 7.55 4.24" />
  <path d="M3.29 7 12 12m0 0 8.71-5M12 12v10" />
  <path d="m17 13 5 5m-5 0 5-5" />
</svg>

```

File: icons/illustrator_x.svg [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<svg
```
Comment (ericfennis) [inline icons/illustrator_x.svg:1]: I think this file shouldn't be here. Probably copy of `package-x`

**Review by ericfennis — CHANGES_REQUESTED** [review]
The svgs shouldn't contain any "combined" paths. This renders not well in design programs like figma.
**Screenshot:**
image

I added some suggestion to fix it!

**Pull request merged**